spec/SpecRunner.html

```diff
@@ -77,8 +77,8 @@
   <script src="spec/marker.js"></script>
   <script src="spec/mask.js"></script>
   <script src="spec/matrix.js"></script>
-  <script src="spec/memory.js"></script>
   <script src="spec/number.js"></script>
+  <script src="spec/nested.js"></script>
```

Comment:
this has to go. nested was removed

Reply:
Oh okay. Weird it didn't error on that. No issues

Reply:
done.

Reply:
jeah its just a not found script file. it gives a network error and that was it^^